pkgs/by-name/tp/tproxy/package.nix

```
@@ -0,0 +1,32 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+}:
+
+buildGoModule rec {
+  pname = "tproxy";
+  version = "0.8.0";
+
+  src = fetchFromGitHub {
+    owner = "kevwan";
+    repo = "tproxy";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-d4ZijF3clu00WZQGlurTkGkedurjt9fqfShdjbZWCSI=";
+  };
+
+  vendorHash = "sha256-tnVzX0crDdkRND7Au0CaTdmwLVxVjxU1jxCRutl48S8=";
+
+  ldflags = [
+    "-w"
+    "-s"
+  ];
+
+  meta = with lib; {
+    description = "A cli tool to proxy and analyze TCP connections";
```

```suggestion
    description = "CLI tool to proxy and analyze TCP connections";
```

As per https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#meta-attributes